.github/workflows/build-msvc.yml

```diff
@@ -71,19 +71,49 @@
             C:\cygwin-packages
           key: cygwin-packages
 
+      - name: Get Autoconf cache key
```

Comment:
Suggestion: `Compute a key to cache configure results`

Reply:
Nice, thanks.